integration/kubeflow/fluid_components.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2024 The Fluid Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl, compiler
+
+# Create a Fluid dataset which contains data in S3.
+@dsl.component(packages_to_install=['git+https://github.com/fluid-cloudnative/fluid-client-python.git'])
```

Comment:
Actually, we should avoid use `package_to_install` in production environment ([more details](https://www.kubeflow.org/docs/components/pipelines/v2/components/containerized-python-components/)). But for now we only have image which install Fluid Python SDK in our image repo. I think this image should be maintained by Fluid community, so this maybe need to update when we have a official base image.

Reply:
Sure. We can handle this in next PR. And currently we have optimized the fluid sdk, please check this doc for reference. https://github.com/fluid-cloudnative/fluid-client-python/blob/master/examples/ml_pipeline/pipeline.ipynb